### ISSUE TITLE
split into multiple files?

### ISSUE THREAD

**Issue by lenscas** [SomeoneElse37/BattleBot#6]
Currently battlebot.py is pretty large. Maybe we should give some things their own file?
I personally was thinking of moving the help dictionary to its own file(s) as it is currently spanning from line 1713 to 2062.
Some other candidates are
- the various classes that we use. (They are classes, and some kind of big)
- the getReply function (A lot less of a problem then some other things but also kind of big)
- the fetching and writing to the database (Makes it easier to switch databases if we need/want to)
- other things?

Should or shouldn't we do this? Also, if we do start to use multiple files are there other things that need their own file?

**Comment by SomeoneElse37** [comment]
Yeah, probably not a bad idea. The main reason I didn't do anything like this in the past is that I wasn't entirely clear on how multiple-file programs in Python work. I know in Java, you normally just put each class in its own file, but that's not so much the case in Python (at least, according to the book I learned Python from); and in C/C++, including a header you wrote and including a header in the standard library use different syntax, so I just didn't bother.
